core/worker/src/main/java/com/cognifide/aet/worker/drivers/chrome/ChromeCommunicationWrapperImpl.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Automated Exploratory Tests <p> Copyright (C) 2013 Cognifide Limited <p> Licensed under the
+ * Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at <p> http://www.apache.org/licenses/LICENSE-2.0
+ * <p> Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cognifide.aet.worker.drivers.chrome;
+
+import java.util.Set;
```

Comment:
Please make sure you have properly imported [AET code style](https://github.com/Cognifide/aet/blob/master/CONTRIBUTING.md#coding-conventions),